.asf.yaml

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+github:
+  description: "Apache Ignite"
+  homepage: https://ignite.apache.org/
+  labels:
+    - distributed-sql-database
+    - iot
+    - osgi
+    - network-client
+    - ignite
+    - data-management-platform
+    - big-data
+    - cloud
+    - database
+    - network-server
+    - hadoop
```

Comment:
Do we remove Hadoop support from Ignite?

Reply:
These are the current labels of Apache Ignite repository on github

Reply:
I'm not sure we can remove this label without discussion.

IGFS and Ignite Hadoop Accelerator were discontinued, but Ignite can still be used to speed up Hadoop, as described on this page: https://ignite.apache.org/use-cases/hadoop-acceleration.html

Reply:
Maxim, I would also add the following labels:

- cache
- in-memory computing
- in-memory database